### PR TITLE
fix(react-scheduler): render TimeScale of the same size as TimeTable

### LINKED ISSUE #2401: Adds extra empty cell if the cell's count is even
```jsx
<DayView
  startDayHour={23}
  endDayHour={24}
  cellDuration={20}
/>
```
image

### PR DESCRIPTION
**Pull request by AryamnovEugeniy**
Fixes #2401
BREAKING CHANGE:

The `DayView` and `WeekView` plugins now do not have the `timeScaleRowComponent` and `timeScaleCellComponent` properties. Instead, they have a new, `timeScaleLabelComponent`, property.

```diff
...
<WeekView
-  timeScaleRowComponent
-  timeScaleCellComponent
+  timeScaleLabelComponent
/>
<DayView
-  timeScaleRowComponent
-  timeScaleCellComponent
+  timeScaleLabelComponent
/>
...
```

### CODE REVIEW

**Comment by MaximKudriavtsev** [comment]
We should not forget to write a BC here 🙄

**Comment by MaximKudriavtsev** [comment]
We should fix this displaying issue. Our site demos have 850px width.
image

As a resolve, we can change the font size `0.75rem -> 0.7rem`.

File: packages/dx-react-scheduler-material-ui/src/templates/views/vertical/time-scale/label.test.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { createShallow, getClasses } from '@material-ui/core/test-utils';
+import { Label } from './label';
+
+describe('Vertical view TimePanel', () => {
```
Comment (MaximKudriavtsev) [inline packages/dx-react-scheduler-material-ui/src/templates/views/vertical/time-scale/label.test.jsx]: Vartical view TimeScale

File: packages/dx-react-scheduler-material-ui/src/templates/views/vertical/time-scale/layout.jsx [hunk 2]
```diff
@@ -1,56 +1,57 @@
...
+  ticks: {
+    width: spacing(1),
+  },
+});
 
 const LayoutBase = ({
-  cellComponent: Cell,
+  labelComponent: Label,
   rowComponent: Row,
+  tickCellComponent: TickCell,
```
Comment (MaximKudriavtsev) [inline packages/dx-react-scheduler-material-ui/src/templates/views/vertical/time-scale/layout.jsx]: We shouldn't use object destructuring alias here

File: packages/dx-react-scheduler/docs/reference/week-view.md [hunk 3]
```diff
@@ -77,18 +76,16 @@ Describes properties passed to a component that renders a time scale layout.
...
+### WeekView.TimeScaleLabelProps
 
-Describes properties passed to a component that renders a time scale cell.
+Describes properties passed to a component that renders a time scale label.
 
 Field | Type | Description
 ------|------|------------
-endDate | Date | Specifies the cell end time.
-startDate? | Date | Specifies the cell start time.
+time? | Date | Specifies the cell's time.
```
Comment (MaximKudriavtsev) [inline packages/dx-react-scheduler/docs/reference/week-view.md]: According to the ts definitions `time` is required property.

**Comment by AryamnovEugeniy** [comment]
FPR

**Comment by AryamnovEugeniy** [comment]
FPR

**Pull request merged**